substrate/frame/revive/fixtures/build.rs

```diff
@@ -61,19 +61,25 @@ impl Entry {
 	}
 }
 
-/// Collect all contract entries from the given source directory.
-fn collect_entries(contracts_dir: &Path) -> Vec<Entry> {
-	fs::read_dir(contracts_dir)
-		.expect("src dir exists; qed")
-		.filter_map(|file| {
+/// Collects all contract entries from the given source directory.
+///
+/// Returns a list of all rust contracts that need to be compiled to PVM
+/// and a list of all contracts already in PVM bytecode which just need
+/// to be copied.
+fn collect_entries(contracts_dir: &Path) -> (Vec<Entry>, Vec<String>) {
```

Comment:
why do we need this change can't you just use include_bytes! macro?

Reply:
I wanted to reuse the `compile_module` function that was being used in fixtures. I guess I can just use `include_bytes!` and in the future we include solidity files here which we want to do anyway